lambdas/raster_tiler/lambda_function.py

```diff
@@ -236,17 +236,21 @@ def get_tile_array(src_tile: str, window: Window) -> np.ndarray:
 
     logger.debug("Get Tile Array")
 
-    gdal_env = {}
+    # if running lambda in localstack, need to use special docker IP address
+    # provided in env to reach localstack
     if AWS_ENDPOINT_HOST:
         gdal_env = {
             "AWS_HTTPS": "NO",
             "AWS_VIRTUAL_HOSTING": False,
             "AWS_S3_ENDPOINT": AWS_ENDPOINT_HOST,
             "GDAL_DISABLE_READDIR_ON_OPEN": "NO",
         }
+    else:
+        gdal_env = {
+            "GDAL_DISABLE_READDIR_ON_OPEN": "EMPTY_DIR",
```

Comment:
I always forget, what does this do again?

Reply:
Devseed has a good description: https://developmentseed.org/titiler/advanced/performance_tuning/#gdal_disable_readdir_on_open.  Do we know if our datasets have supplemental files like overlays or not?

Reply:
Also, it seems to say that the options are `EMPTY_DIR`  or `FALSE`, but we use `NO`, might want to verify if that works as expected (might default to `FALSE` anyways).